sparkmagic/sparkmagic/controllerwidget/manageendpointwidget.py

```diff
@@ -122,3 +127,12 @@ def get_info_endpoint_widget(self, endpoint, url):
             text = "No sessions on this endpoint."
 
         return self.ipywidget_factory.get_html(text, width=width)
+
+    def initialize_kerberos(self, endpoint):
```

Comment:
What about getting the ticket directly from `ReliableHttpClient`? Then, upon creation of the object, the ticket will be obtained (and renewed when needed) and when the client is not to be used anymore, the client would also call `kdestroy`.

We could probably achieve this by creating a `start` and `stop` method on `ReliableHttpClient`. We can also call those methods from `__enter__` and `__exit__` so that the object can also be used with a `with` clause.